saleor/channel/migrations/0020_migrate_checkout_ttl_release_funds.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.2.25 on 2025-06-25 10:48
+
+from django.conf import settings
+from django.db import migrations, transaction
+from django.db.models import Q
+
+BATCH_SIZE = 5000
+
+
+def queryset_in_batches(queryset):
+    """Slice a queryset into batches.
+
+    Input queryset should be sorted be pk.
+    """
+    start_pk = 0
+
+    while True:
+        qs = queryset.filter(pk__gt=start_pk)[:BATCH_SIZE]
+        pks = list(qs.values_list("pk", flat=True))
+
+        if not pks:
+            break
+
+        yield pks
+
+        start_pk = pks[-1]
+
+
+def migrate_env_variable_setting_to_channels(apps, _schema_editor):
+    Channel = apps.get_model("channel", "Channel")
+    turn_on = int(settings.TRANSACTION_BATCH_FOR_RELEASING_FUNDS) > 0
+    channels = Channel.objects.order_by("pk").filter(
+        ~Q(
+            checkout_ttl_before_releasing_funds=settings.CHECKOUT_TTL_BEFORE_RELEASING_FUNDS
+        )
+        | ~Q(release_funds_for_expired_checkouts=turn_on)
+    )
+    ids = channels.values_list("pk", flat=True)[:BATCH_SIZE]
```

Comment:
The update will be done only once for the `BATCH_SIZE`. `queryset_in_batches` is not used